srcpkgs/python3-openai/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'python3-openai'
+pkgname=python3-openai
+version=0.25.0
+revision=1
+build_style=python3-pep517
+hostmakedepends="python3-setuptools python3-wheel"
+depends="python3-requests python3-numpy python3-pandas python3-tqdm python3-pandas-stubs python3-openpyxl"
```

Comment:
```suggestion
depends="python3-requests python3-numpy python3-pandas python3-tqdm python3-pandas-stubs python3-openpyxl"
checkdepends="python3-pytest"
```

Reply:
This looks to be easy enough although it will fail on the build server unless it has a valid API key set as an environment variable.

Reply:
in that case, mark tests as extended